### ISSUE TITLE
Add Analog Output

### ISSUE THREAD

**Issue by brentru** [adafruit/Wippersnapper_Protobuf#37]
Add analog output functionality (Cloud2Device)

Address after https://github.com/adafruit/Wippersnapper_Protobuf/issues/36

Misc notes:
```* Analog Output 
    * Loren: Mini-Slider
    * Same as PWM signal 
    * PWM Outputs
        * Two settings
            * Frequency 
                * Look at what arduino and blynk use
                * OK with set # of frequencies they can pick from
                    * 1mhz 
                    * LEDs 10khz
                    * Servos: 50khz
            * Duty cycle 
                * %’s 
                    * Slider element on modal?
* SERVOS
    * Separate device
        * 50Hz
            * Slider only goes between 1 and 2 %
* Do generic LED PWM 
* ESP32 different : ledCWrite, LEDCmanage
* Generic arduino: analogWrite
* > 3 PWMs, show an error on the frontend
    * Device needs to return an error
* Proto:
    * Pin
    * Freq
    * Min duty cycle
        * LED: 0 
        * Servo: 1ms
    * Max duty cycle
        * LED: 100 ms
        * Servo: 2ms
    * NOTE: Min and Max define the “slider” UI element
        * Feed value is the % slider, scaled by the end device
            * 50hz slider, min max, duty cycle
                * Device does the math
    * Duty cycle within that scaling 
* Servo is subset of PWM
* ````

**Comment by brentru** [comment]
This should now be possible with the work done in https://github.com/adafruit/Wippersnapper_Protobuf/issues/91 and the ws_esp32_ledc management class

**Comment by brentru** [comment]
Merged in #100
